.github/workflows/ci-build.yml

```diff
@@ -153,15 +157,15 @@ jobs:
     steps:
     - uses: actions/checkout@v3
       with:
-        ref: ${{ github.event.pull_request.head.ref || github.ref }}
+        ref: ${{ inputs.REF || github.ref }}
```

Comment:
why is the github.event.pull_request.head.ref being removed though? From the description I thought are only trying to add `inputs.REF`?

Reply:
looking at this more, maybe we don't need an input? What about?

```suggestion
        ref: ${{ github.head_ref || github.ref }}
```

Reply:
these workflow are triggered by `workflow_run` event in here `github.head_ref` doesn't exist.

Reply:
or `github.events.pull_request.head.ref`

Reply:
took me a while to realise that in here triggering event is `workflow_run` too.

Reply:
ok, thanks for clarifying.